docs/fern/scripts/check_published_styles.py

```
@@ -0,0 +1,90 @@
+    ("", ".dynamo-welcome__terminal", "LandingStyles"),
+    ("community", ".dynamo-community-page", "LandingStyles"),
+    ("digest", ".dynamo-blog-art__grid", "BlogStyles"),
+    ("reference/compatibility", ".dynref-panel", "ReferenceStyles"),
+]
+
+
+def fetch(url: str, retries: int, delay: float) -> str:
+    last: Exception | None = None
+    for attempt in range(1, retries + 1):
+        try:
+            with urllib.request.urlopen(url, timeout=60) as response:
```

Ruff (0.16.0)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)